client/src/components/Workflow/Editor/Index.vue

```diff
@@ -42,8 +42,12 @@
             <div class="unified-panel-header" unselectable="on">
                 <div class="unified-panel-header-inner">
                     <span class="sr-only">Workflow Editor</span>
-                    <span v-if="!isNewTempWorkflow || name">{{ name }}</span>
-                    <i v-else>Create New Workflow</i>
+                    <span>
+                        {{ name || "..." }}
+                        <i v-if="isNewTempWorkflow">
+                            (Click Save <span class="fa fa-floppy-o" /> to create this workflow)
+                        </i>
```

Comment:
I'd rather we hide the implementation details from the user, and stick with the "create" -> "save" wording, even if it doesn't reflect what the code is doing in the background.

I do not think a user needs to know if the workflow is actually created on the backed, or the front-end, and making this differentiation seems like it may just lead to confusion.

Rather, I'd argue for keeping the wording for "create" outside of the workflow editor, and always calling the save-button "save", even if it creates the workflow on the back-end.

Think about how other software does this. For example, when you create a new office document (word, excel, etc.) there isn't actually any new document created on disk. Instead the first save creates the actual document. The actions are still called "create" and "save" though. This is a fairly standard pattern users are used to.

Reply:
+1, create outside, save inside (regardless of state) -- I think this is where the discussion ended up on the call last week, too.

Reply:
> Think about how other software does this.

That's exactly why I initially left it as Save when I made the change back then. But maybe I misunderstood the consensus in the meeting. Ok, I'll still keep the word "Save" inside the editor. Thanks!